src/pages/hegel/reference/mechanical-object/first-paragraph.mdx

```diff
@@ -1,17 +1,33 @@
+What exactly are these moments of the `mechanical object` that have become an immediate identity? Hegel clarifies this in the following sentence:
 
-The moments of the “mechanical object” are the determinations of the “Concept” : “universal”, “particular”, and “individual”. It is these moments that are immediately identical to each other. In the “mechanical object”, the “universal” is immediately the “particular” and the “individual”. In other words, the “universal” is not a universal that has the basic essence of a thing and that finds its essence instantiated in particular and individual objects. It is not, for example, like the universal concept of a chair that states that a chair must be ‘so and so’ and that serves as the essence of armchairs and swivel chairs, alike. It is not, as Hegel writes, a universal “in the sense of a community of properties” (Miller, 711).  Rather, it is a universal that is immediately identical to the particular and the individual. In other words, the general concept of a chair is identical to all particular and individual chairs – there are not some chairs that are short and some that are long, or some chairs that offer good lumbar support and others that do not, rather, all chairs are identical to each other. The moments of the “mechanical object”, then, are treated as identical to each other, there is no distinction in conceiving of the “mechanical object” as universal or as individual.
+“It is therefore in and for itself a universal - universality not in the sense of a community of properties, but a universality that pervades the particularity and in it is immediate individuality” (Hegel, 1991, 711).  
 
+The moments of the `mechanical object` are the determinations of the `Concept` : `universal`, `particular`, and `individual`. It is these moments that are immediately identical to each other. In the `mechanical object`, the `universal` is immediately the `particular and the `individual`. In other words, the `universal` is not a universal that has the basic essence of a thing and that finds its essence instantiated in particular and individual objects. It is not, for example, like the universal concept of a chair that states that a chair must be ‘so and so’ and that serves as the essence of armchairs and swivel chairs, alike. It is not, as Hegel writes, a universal “in the sense of a community of properties” (Hegel, 1991, 711).  Rather, it is a universal that is immediately identical to the particular and the individual. In other words, the general concept of a chair is identical to all particular and individual chairs – there are not some chairs that are short and some that are long, or some chairs that offer good lumbar support and others that do not, rather, all chairs are identical to each other. The moments of the `mechanical object`, then, are treated as identical to each other, there is no distinction in conceiving of the `mechanical object` as universal or as individual.
```

Comment:
There is a bug here. Probably an extra backtick skulking around somewhere or one missing. 

![image](https://github.com/user-attachments/assets/36bd96e4-5a78-430f-a91f-9734303c47ca)

Reply:
Also, single right quotation marks around "so and so", change these to apostrophes.

Reply:
Done.